modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -389,6 +403,10 @@ export default function ViewObjectDefinitions({
 					objectFolderExternalReferenceCode={
 						selectedObjectFolder.externalReferenceCode
 					}
+					onAfterSubmit={() => {
+						setReloadFDS(true);
+					}}
+					reload={false}
```

Comment:
Hey @Artartur, this `reload` prop is no longer needed as we removed all cases where this would be `true`, so you can remove this prop and the condition that uses it inside the modal. :smile:

Reply:
Done!